server/webhook/tag.go

```diff
@@ -39,7 +39,7 @@ func (w *webhook) handleDMTag(event *gitlab.TagEvent) ([]*HandleWebhook, error)
 }
 
 func (w *webhook) handleChannelTag(event *gitlab.TagEvent) ([]*HandleWebhook, error) {
-	senderGitlabUsername := event.UserName
+	senderGitlabUsername := w.gitlabRetreiver.GetUsernameByID(event.UserID)
```

Comment:
Is this needed because `event.UserUsername` does not come in the message?

Reply:
Yes, it is :) See [here](https://github.com/xanzy/go-gitlab/blob/master/event_webhook_types.go#L86) or [here](https://docs.gitlab.com/ce/user/project/integrations/webhooks.html#tag-events)